app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -89,14 +88,8 @@ class RoutePresenter(
         }.asOptional()
     }
 
-    private fun accountToDataStoreActions(it: Pair<Optional<SyncCredentials>, State>): Optional<DataStoreAction> {
-        val (opt, state) = it
-
-        val credentials = opt.value ?: return DataStoreAction.Reset.asOptional()
-
-        if (state != State.Unprepared) {
-            return Optional(null)
-        }
+    private fun accountToDataStoreActions(optCredentials: Optional<SyncCredentials>): Optional<DataStoreAction> {
+        val credentials = optCredentials.value ?: return DataStoreAction.Reset.asOptional()
```

Comment:
We get a `null` `SyncCredentials` when we log out don't we?

Could you stick a comment in here just to say that a) on first run we get `null` credentials and b) on log out?